pkg/utils/crtl_utils.go

```diff
@@ -51,9 +51,8 @@ func RequeueIfError(err error) (ctrl.Result, error) {
 	return ctrl.Result{}, err
 }
 
-// Helper function which requeues immediately if the object generation has not changed.
-// Otherwise, since the generation change will trigger an immediate update anyways, this
-// will not requeue.
+// RequeueImmediatelyUnlessGenerationChanged which requeues immediately if the object generation has not changed.
```

Comment:
RequeueImmediatelyUnlessGenerationChanged which requeues immediately -> RequeueImmediatelyUnlessGenerationChanged requeues immediately

Reply:
I have revised the comment.